imagej/imagej-ops2/src/main/java/net/imagej/ops2/threshold/localPhansalkar/ComputeLocalPhansalkarThreshold.java

```diff
@@ -90,7 +91,7 @@
 	 */
 	@Override
 	public void compute(final Iterable<T> inputNeighborhood,
-		final T inputCenterPixel, final Double k, final Double r,
+		final T inputCenterPixel, @Optional final Double k, @Optional final Double r,
```

Comment:
We could just inline the other `compute` method, looks like it's not being called anywhere?

The other method is `public static`, but it isn't being exported...

Reply:
Agreed